src/test/java/org/folio/oaipmh/helpers/records/RecordMetadataManagerTest.java

```diff
@@ -364,7 +364,7 @@ private String getJsonObjectFromFile(String path) {
       byte[] encoded = Files.readAllBytes(Paths.get(file.getPath()));
       return new String(encoded, StandardCharsets.UTF_8);
     } catch (IOException e) {
-      logger.error("Unexpected error", e);
+      logger.error("Unexpected error.", e);
```

Comment:
Is it really unexpected assuming that we caught a specific exception (IOException)

Reply:
Message changed to `logger.error("Error occurred while reading the file \"{}\".", path, e);`